fastlane/lib/fastlane/actions/latest_testflight_build_number.rb

```diff
@@ -72,7 +78,7 @@ def self.available_options
                                        description: "The version number whose latest build number we want",
                                        optional: true),
           FastlaneCore::ConfigItem.new(key: :initial_build_number,
-                                       env_name: "INTITIAL_BUILD_NUMBER",
+                                       env_name: "INITIAL_BUILD_NUMBER",
```

Comment:
Just corrected that because it seemed a spelling error. A quick grep shows this is the only action on sources that uses the parameter and environment variable.

It would only break a working setup in case if the mispelled variable is used, and only in that action.

Mispelled variable, prior to commit.
`grep -R INTITIAL_BUILD_NUMBER *
fastlane/lib/fastlane/actions/latest_testflight_build_number.rb:                                       env_name: "INTITIAL_BUILD_NUMBER",`


Corrected variable,
`grep -R INITIAL_BUILD_NUMBER *
fastlane/lib/fastlane/actions/latest_testflight_build_number.rb:                                       env_name: "INITIAL_BUILD_NUMBER",`

Parameter "initial_build_number"

`grep -R initial_build_number *
fastlane/lib/fastlane/actions/latest_testflight_build_number.rb:          UI.user_error!("could not find a build on iTC - and 'initial_build_number' option is not set") unless params[:initial_build_number]
fastlane/lib/fastlane/actions/latest_testflight_build_number.rb:          build_number = params[:initial_build_number]
fastlane/lib/fastlane/actions/latest_testflight_build_number.rb:          FastlaneCore::ConfigItem.new(key: :initial_build_number,`


@KrauseFx You mean setting a default value , and making the parameter non-optional, like this?

`          FastlaneCore::ConfigItem.new(key: :initial_build_number,
                                       env_name: "INITIAL_BUILD_NUMBER",
                                       description: "sets the build number to given value if no build is in current train",
                                      default_value: 1,
                                       is_string: false)`